src/implementations/twig/components/file/file.html.twig

```diff
@@ -4,7 +4,7 @@
   Parameters:
   - "variant" (string) (default: 'default') Name of the variant [default, thumbnail]
```

Comment:
we are still passing the `variant` parameter to the other twig files, but I think that it is no longer used there